tensorboard/webapp/widgets/data_table/header_cell_component.ng.html

```diff
@@ -44,14 +49,13 @@
           "
       svgIcon="arrow_downward_24px"
     ></mat-icon>
-  </div>
-  <div
+  </button>
+  <button
+    mat-icon-button
     *ngIf="(header.removable || header.sortable) && !disableContextMenu"
     class="context-menu-container show-on-hover"
+    (click)="onContextMenuOpened($event)"
```

Comment:
When I press "space" on this context menu, the context menu is actually opened in the top-left corner of the runs table rather than next to the location of the button.

![image](https://github.com/tensorflow/tensorboard/assets/17152369/83bf73e1-900b-4253-b208-2ab458e921c8)

Is there any way to fix this? Maybe take into account the location of the button when opening the context menu?

Reply:
Yea I noticed this too I added a task to fix it in our task list.